### PR TITLE
Update acknowledgements in README

### PR DESCRIPTION
**Pull request by atomb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,17 +205,23 @@ and by cryptographers to
...
+Foner, Kevin Quick, Kyle Carter, Ledah Casburn, Lee Pike, Levent Erkök,
+Lisanna Dettwyler, Magnus Carlsson, Mark Shields, Mark Tullsen, Matt
+Sottile, Nathan Collins, Philip Weaver, Robert Dockins, Sally Browning,
+Sam Anklesaria, Sigbjørn Finne, Stephen Magill, Thomas Nordin, Trevor
+Elliott, and Tristan Ravitch.
+
+Much of the work on Cryptol has been funded by, and lots of design input
+was provided by the team at the [NSA's Trusted Systems Research
+Group](http://www.nsa.gov/research/ia_research/), including Brad Martin,
+Frank Taylor and Sean Weaver.
```
Comment (weaversa) [inline README.md:224]: Oxford comma or not?

File: README.md [hunk 2]
```diff
@@ -205,17 +205,23 @@ and by cryptographers to
...
+Jones, Ben Selfridge, Brian Huffman, Brian Ledger, David Thrane
+Christiansen, David Lazar, Dylan McNamee, Eddy Westbrook, Edward Yang,
+Eric Mertens, Eric Mullen, Fergus Henderson, Iavor Diatchki, Jeff Lewis,
+Jim Teisher, Joe Hendrix, Joe Hurd, Joe Kiniry, Joel Stanley, Joey
+Dodds, John Launchbury, John Matthews, Jonathan Daugherty, Kenneth
+Foner, Kevin Quick, Kyle Carter, Ledah Casburn, Lee Pike, Levent Erkök,
+Lisanna Dettwyler, Magnus Carlsson, Mark Shields, Mark Tullsen, Matt
+Sottile, Nathan Collins, Philip Weaver, Robert Dockins, Sally Browning,
+Sam Anklesaria, Sigbjørn Finne, Stephen Magill, Thomas Nordin, Trevor
+Elliott, and Tristan Ravitch.
```
Comment (weaversa) [inline README.md:219]: Oxford comma here

File: README.md [hunk 3]
```diff
@@ -205,17 +205,23 @@ and by cryptographers to
...
+
+Much of the work on Cryptol has been funded by, and lots of design input
+was provided by the team at the [NSA's Trusted Systems Research
+Group](http://www.nsa.gov/research/ia_research/), including Brad Martin,
```
Comment (weaversa) [inline README.md:223]: Also, the affiliation changed. It was correct as Laboratory for Advanced Cyber Security Research.

File: README.md (hunk 1 above)
Comment (atomb) [reply README.md:225]: Oh, yes!

File: README.md (hunk 3 above)
Comment (atomb) [reply README.md]: Undone back to the right one.

**Review by kquick — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -205,15 +205,22 @@ and by cryptographers to
...
+Weakly, Jeff Lewis, Jim Teisher, Joe Hendrix, Joe Hurd, Joe Kiniry, Joel
+Stanley, Joey Dodds, John Launchbury, John Matthews, Jonathan Daugherty,
+Kenneth Foner, Kevin Quick, Kyle Carter, Ledah Casburn, Lee Pike, Levent
+Erkök, Lisanna Dettwyler, Magnus Carlsson, Mark Shields, Mark Tullsen,
+Matt Sottile, Nathan Collins, Philip Weaver, Robert Dockins, Ryan Scott,
+Sally Browning, Sam Anklesaria, Sigbjørn Finne, Stephen Magill, Thomas
+Nordin, Trevor Elliott, and Tristan Ravitch.
+
+Much of the work on Cryptol has been funded by, and lots of design input
+was provided by, the team at the [Laboratory for Advanced Cyber Security
```
Comment (weaversa) [inline README.md:223]: "NSA's Laboratory for Advanced Cybersecurity Research"

Cybersecurity is one word :-)

**Pull request merged**